TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -94,6 +113,102 @@ async Task<Result> SetPaymentTime(int memberId)
     }
 
 
+    public Task<Result<List<TransactionResponse>>> Get(MemberContext memberContext, int facilityId, int skip, int top,
+        TransactionFilterProperty filterProperty, CancellationToken cancellationToken = default)
+    {
+        return Validate()
+            .Bind(GetTransactions);
+
+
+        Result Validate()
+        {
+            var validator = new FacilityRequestValidator(memberContext, _context);
+            var validationResult = validator.ValidateGetOrUpdate(new FacilityRequest(facilityId, memberContext.AccountId));
+            return validationResult.ToResult();
+        }
+
+
+        async Task<Result<List<TransactionResponse>>> GetTransactions()
+        {
+            var query = _context.Transactions.Where(t => t.FacilityId == facilityId);
+
+            query = filterProperty switch
+            {
+                TransactionFilterProperty.CreatedASC => query.OrderBy(t => t.Created),
+                TransactionFilterProperty.AmountASC => query.OrderBy(t => t.Amount),
+                TransactionFilterProperty.AmountDESC => query.OrderByDescending(t => t.Amount),
```

Comment:
Could you decapitalize these to `Asc` and `Desc` according to our naming guidelines?